Sophia Sun Project Pitch

```diff
@@ -0,0 +1,244 @@
+
+I think my newest idea is the right way to interact with the problem domain because when musicians look up the chords for a song, they want to be able to play the music right away, and I want my program to be as hassle-free as possible. I think copying the chord names from an existing chord sheet is relatively easy to do. Also, this input file would be a txt file, so the user doesn't have to download anything special to input the chord names. Also, copy/pasting in the name of the chord requires no understanding of music theory, which I do not expect for my users to have. 
+
+
+### Operation (semantics)
+# What might happen when a program runs? 
+
+The parser needs to break up each line of the text file into a separate String. I think parsing might be really tricky because there are different ways to express a chord with the same root, quality, and inversion. The program needs to break up the input into several features: the root of the chord, quality, and inversion.
+
+I will have some sort of array that contains all the possible notes in a chord. The array starts with A, and each note is a half step apart: 
+A A# Ab B B# Bb C C# Cb D D# Db E Eb F F# G G# Gb 
+Index 0: A
+Index 1: A# / Bb
+Index 2: B 
+Index 3: B# / C 
+Index 4: C# / Db
+Index 5: D 
+Index 6: D# / Eb
+Index 7: E
+Index 8: F
+Index 9: F# / Gb
+Index 10: G
+Index 11: G# / Ab
+
+My program takes the root of the chord and the quality of the chord and finds the notes that are in this chord. If the input specifies the root but not the quality of the chord, then I know that a lower case root represents a minor chord with that root, and an upper case root represents a major chord with that root. For example, that means that the input string BMin is the same chord as b. 
+
+All chords of the same quality (Major, minor, diminished, augmented, diminished 7th, Dominant 7th, Major 7th, etc) are composed of notes that are the same interval apart. For example, an A major chord is A C# E, and a C major chord is C E G. As you can see from the array above, 
+A = index 0, 
+C# = index 4, 
+E = index 7, 
+and 
+C = index 3,
+E = index 7, 
+G = index 10. 
+
+As you can tell, the difference between the first note and the second note is 7-3 = 4-0 = 4 half steps.
+The difference between the second note and the third note is 10-7 = 7-4 = 3 half steps. I will tell my program what set of intervals are between the different notes depending on the quality of the chord, and it will return the notes that are in the chord!!
+Then, the program would look at what inversion was specified (and if there was no inversion, assume a root position of the chord), and rearrange the order of the notes in the chord accordingly.
+
+Then the program compiles my DSL into a LilyPond program/code, 
```

Comment:
I was worried that this would be a difficult step, but I looked up LilyPond at it looks super user-friendly. It looks like a great tool, especially for this project.